lib/typescript.js

```diff
@@ -23,6 +18,7 @@ export default [
       "@typescript-eslint": typescript,
     },
     rules: {
+      "no-undef": "off",
```

Comment:
Is this correct?

Reply:
Yep, so when we are using typescript your should turn off eslints default no-undef. I believe typescript has its own one that is used instead and shows Type things like `Partial` and `Omit`